cmd/out/main.go

```diff
@@ -117,6 +118,26 @@ func main() {
 
 	logrus.Info("pushed")
 
+	if req.Source.ContentTrust.Enable {
```

Comment:
Rather than having an `enable` field maybe we can make `ContentTrust` a pointer and just check if it's non-nil. If a user configures any of those fields they probably want it enabled.

Although, one reason to have an explicit `Enable` field is if all the other fields are optional - in which case the only thing to do would be to enable it. Is that possible? It seems like the server field is required at least.

Reply:
Yeah, `server` field is optional because there are default Notary server `https://notary.docker.io`. But AFAIK, at least `repository_key_id` and `repository_key` are not optional. So remove `enable` field sounds good choice.